pages/api/sendgrid.ts

```diff
@@ -27,7 +27,7 @@ export default async function handler(
       const msg = {
         to: TO_ADDRESS, // Change to your recipient
         from: FROM_ADDRESS, // Change to your verified sender
-        subject: `OpenSats Application for ${req.body.project_name}`,
+        subject: `MAGIC Monero Fund Application for ${req.body.project_name}`,
```

Comment:
This part of the app is inactive since we don't have a sendgrid account, but it's OK to change now and revisit sendgrid functionality later.